src/vscode-bicep/src/commands/decompile.ts

```diff
@@ -128,9 +132,14 @@ export class DecompileCommand implements Command {
     );
   }
 
-  public static async mightBeArmTemplate(documentUri: Uri): Promise<boolean> {
+  public static async mightBeArmTemplateNoThrow(
```

Comment:
The "Uri or string" contract could be misinterpreted as you could accidentally pass a Uri in string format - worth breaking into two functions?

Reply:
good point